air-script/Cargo.toml

```diff
@@ -1,19 +1,28 @@
 [package]
 name = "air-script"
 version = "0.1.0"
+description="AirScript language compiler"
+authors = ["miden contributors"]
+readme="README.md"
+license = "MIT"
+repository = "https://github.com/0xPolygonMiden/air-script"
+documnetation = "https://0xpolygonmiden.github.io/air-script/"
+categories = ["compilers", "cryptography"]
+keywords = ["air", "stark", "zkp"]
```

Comment:
should we also have "zero knowledge" or "zero knowledge proofs" instead of just "zkp"?